src/components/modals/Modal/Modal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import style from './modal.module.css'
+import Backdrop from '../../back-drop/back-drop'
+import { Lift, useLifts } from '../../../context/lift-context/lift-context'
+import Select from '../../ui/select/Select'
+import Lists from '../../dash-board/Lists/Lists'
+import Button from '../../ui/button/Button'
+
+const Modal: React.FC = () => {
+  const { filteredArr, selectedItemId, isOpen, onShow, onUpdate, isLoading } =
+    useLifts()
+  const [selectedItemStatus, setSelectedItemStatus] = useState<string | null>(
+    null
+  )
+  let modalData = null
```

Comment:
why using this?? Create a function and return modalData from that function

Reply:
created a component for this instead